solutions/instances/variables.tf

```diff
@@ -202,6 +202,18 @@ variable "scc_instance_tags" {
   default     = []
 }
 
+variable "attach_wp_to_scc_instance" {
```

Comment:
don't expose this - if `provision_scc_workload_protection` is true then the DA code should automatically set attach it to the SCC instance